lsp-mode.el

```diff
@@ -4848,8 +4849,7 @@ When language is nil render as markup if `markdown-mode' is loaded."
         (lsp--render-string value language))
        ((MarkupContent :value :kind)
         (lsp--render-string value kind))
-       ;; plain string
-       ((pred stringp) (lsp--render-string content "markdown"))
+       ((pred stringp) (lsp--render-string content "plaintext"))
```

Comment:
As per spec, when the server is sending simple string it is markdown, so this part should stay as it is:

```
/**
 * MarkedString can be used to render human readable text. It is either a
 * markdown string or a code-block that provides a language and a code snippet.
 * The language identifier is semantically equal to the optional language
 * identifier in fenced code blocks in GitHub issues.
 *
 * The pair of a language and a value is an equivalent to markdown:
 * ```${language}
 * ${value}
 * ```
 *
 * Note that markdown strings will be sanitized - that means html will be
 * escaped.
 *
 * @deprecated use MarkupContent instead.
 */
type MarkedString = string | { language: string; value: string };
```